sharing/process/index.md

```diff
@@ -0,0 +1,53 @@
+---
+tags:
+  - sharing
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+date: 2017-01-26 23:40:06
+title: Submitting Data to the SGID
+categories: []
+---
+
+Thank you for wanting to share your data with the SGID! We've put together a few processes to make sure your datasets are added to the right place. We use [issues](https://guides.github.com/features/issues/) in our [Porter](https://github.com/agrc/porter/issues) repository on GitHub to track where a dataset is at in the process and keep an open and transparent record of everything that happened.
+
+If you're not sure which part of the SGID you should share your data with, check out our sharing [introduction]({% link sharing/index.md %}#join-the-party) for an overview.
+
+If you've still got questions after reading this, {% include contact.html subject="SGID Data Additions" contact=site.data.contacts.sgid text='reach out to ' hide-punctuation=true %}, or anyone else on the [AGRC team]({% link about/contact/index.html %}).
+
+## Contributing Data to an Existing Dataset
+
+If you have data you'd like to add to an existing dataset, please check the data page for the contact info of the AGRC staffer responsible. If none are listed, please reach out via our main email. We are happy to work with authoritative stewards and other qualified individuals to improve our data
+
+## Submitting New Datasets
+
+No matter where your datasets fit best—SGID Index, Open Data, or Open SGID—the process begins with the same first two steps: first, making sure your datasets meet the relevant [qualifications]({% link about/policy/sgid/index.md %}) and then opening an issue in [Porter](https://github.com/agrc/porter/issues) using the relevant template.
+
+Each template includes questions about your data. We'd appreciate it if you fill all these out when you first submit your datasets. Otherwise, the issue will be put on hold until this information is provided.
+
+### SGID Index
+{: .text-left}
+
+It's quite simple to list [qualifying]({% link about/policy/sgid/index.md %}) data in the SGID Index. After you submit an issue in Porter using the "Add data to the SGID Index template, we will review your submission against the qualifications and reach out with any questions. Once we've worked together to resovle any questions or problems, we'll add your dataset to the Index and close the issue.
+
+### Open Data
+{: .text-left}
+
+Adding data to [opendata.gis.utah.gov](https://opendata.gis.utah.gov) requires a few additional steps to navigate the ArcGIS Online sharing mechanics. This time you'll use the "Add data to Open Data" issue template. After the issue is opened, you'll need to request access to the [Open Data Staging Group](https://utah.maps.arcgis.com/home/group.html?id=3d3bd0d238f24f45b2c4b84f1015a317) in ArcGIS Online. This is a staging group that allows us to review the dataset before sharing it through Open Data.
```

Comment:
**question**: have you started to create these new issue templates?

Reply:
Nope, not yet (https://github.com/agrc/porter/issues/88). I intend to do that once the pages are drafted, and then incorporate any needed changes back into the pages. I'll build off your work restructuring the templates.